notebook/static/notebook/js/menubar.js

```diff
@@ -128,6 +128,133 @@ define([
 
         this._new_window(url);
     };
+    
+    MenuBar.prototype._nbconvert_upload_conf = function (download) {
+      
+      var body =  $('<div>');
+      var notebook_path = utils.encode_uri_components(this.notebook.notebook_path);
+
+      var create_json = function(notebook, config){
+        var json_to_pass = {
+          notebook: notebook.toJSON(),
+          config: config
+        };
+        return json_to_pass;
+      };
+
+      
+      var form  = $('<form>');
+      var fileinput = $('<input>')
+        .attr('type', 'file')
+        .attr('tabindex', '1');
+      var fileformat = $('<select>');
+      fileformat.append(
+        $('<option>markdown</option>')
+          .attr('value','markdown')
+      );
+      fileformat.append(
+        $('<option/>')
+          .attr('value','html')
+          .text('html')
+      );
+      fileformat.append(
+        $('<option>rst</option>')
+          .attr('value','rst')
+      );
+      fileformat.append(
+        $('<option>pdf</option>')
+          .attr('value','pdf')
+      );
+      fileformat.append(
+        $('<option>latex</option>')
+          .attr('value','latex')
+      );
+      fileformat.append(
+        $('<option>script</option>')
+          .attr('value','script')
+      );
```

Comment:
done, via loop (which takes a standard structure which will make it easier for linking into by 3rd party exporters)